DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Controllers/RepositoryController.cs

```diff
@@ -83,14 +83,30 @@ public class RepositoryController : BaseApiController
         {
             var token = User.Identity.GetExternalAccessToken();
             var userName = User.Identity.GetUserName();
-
             var gitHubCollaborators = await _gitHubQuery.GetCollaboratorsForRepo(userName, repoName, token);
-
             var gitHubCollaboratorsExceptUser = gitHubCollaborators.Where(collaborator => collaborator.Login != User.Identity.Name);
-
             var users = _securityContextQuery.GetAllUsers();
             var collaboratorsWithUserId = gitHubCollaboratorsExceptUser.AddUserIdToCollaboratorIfExists(users);
 
+            User currentUser = await _userManager.FindByNameAsync(userName);
+            string currentProgectId = currentUser.Claims.Where(el => el.ClaimType == "CurrentProjectId")
+                .Select(el => el.ClaimValue).FirstOrDefault();
+
+            foreach (var item in collaboratorsWithUserId)
```

Comment:
Can I send _securityContextQuery and _userManager in extension method?
Or better create extension methods for _securityContextQuery and _userManager?